### PR TITLE
basic setup of welcome page and routing

### LINKED ISSUE #6: web-welcome page
As a user...
I want to...see the different possible features (links to these features)
So that... I can select a feature to do any specific task

### PR DESCRIPTION
**Pull request by Arefin2023**

### CODE REVIEW

File: web/src/pages/WelcomePage.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.flex-container {
+    display: flex;
+    flex-direction: row;
```
Comment (actyralf) [inline web/src/pages/WelcomePage.css]: `flex-direction: row` is the default, so you can remove this line

File: web/src/pages/WelcomePage.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    flex-wrap: wrap;
+    margin-bottom: 20px;
+}
+.box { 
+        width: 300px;
+        height: 200px;
+        background-color: lightblue;
+        opacity: 90%;
+        border: 1px solid gray;
+        margin: 0,10px;
```
Comment (actyralf) [inline web/src/pages/WelcomePage.css]: this looks like a syntax error. Maybe you meant `margin: 0 10px;`?

**Review by actyralf — CHANGES_REQUESTED** [review]
Hi Arefin,

that looks good to me! Please have a look at my two tiny comments though.

File: web/src/pages/WelcomePage.css (hunk 1 above)
Comment (Arefin2023) [reply web/src/pages/WelcomePage.css]: removed it, thanks

File: web/src/pages/WelcomePage.css (hunk 2 above)
Comment (Arefin2023) [reply web/src/pages/WelcomePage.css]: removed that line, because it doesn't seem to make any difference.

Comment (Arefin2023) [reply web/src/pages/WelcomePage.css]: please check the new commit and approve. thanks

**Review by actyralf — APPROVED** [review]

**Pull request merged**